applications/services/gui/canvas.h

```diff
@@ -146,6 +146,13 @@ void canvas_invert_color(Canvas* canvas);
  */
 void canvas_set_font(Canvas* canvas, Font font);
 
+/** Set custom drawing font
+ *
+ * @param      canvas  Canvas instance
+ * @param      font    Pointer to u8g2 const uint8_t* font array
+ */
+void canvas_set_custom_font(Canvas* canvas, const uint8_t* font);
```

Comment:
Please, rename to `canvas_set_custom_u8g2_font`.

Reply:
done